Removes a native test in RCall that uses GPU and therefore fails to install on machines without GPU.
Author: Orient Kružík (INUITS)
diff -ruN RCall_orig/test/basic.jl RCall/test/basic.jl
--- RCall_orig/test/basic.jl	2023-08-30 03:06:12.000000000 +0200
+++ RCall/test/basic.jl	2023-10-11 18:54:35.664758000 +0200
@@ -96,17 +96,17 @@
 @test_throws Exception t1[:x] = "a"
 
 # graphics
-RCall.rgui_init()
-let f = tempname()
-  rcall(:png,f)
-  rcall(:plot,1:10)
-  rcall(Symbol("dev.off"))
-  @test isfile(f)
-  rm(f)
-  @test !RCall.rgui_start(true)
-  @test_throws ErrorException RCall.rgui_start()
-  @test RCall.rgui_stop()
-end
+# RCall.rgui_init()
+# let f = tempname()
+#   rcall(:png,f)
+#   rcall(:plot,1:10)
+#   rcall(Symbol("dev.off"))
+#   @test isfile(f)
+#   rm(f)
+#   @test !RCall.rgui_start(true)
+#   @test_throws ErrorException RCall.rgui_start()
+#   @test RCall.rgui_stop()
+# end
 
 # S4 rprint
 @test occursin("An object of class",
